hazelcast/src/main/java/com/hazelcast/query/impl/Index.java

```diff
@@ -118,43 +118,6 @@ void putEntry(CachedQueryEntry newEntry, CachedQueryEntry oldEntry, QueryableEnt
      */
     Set<QueryableEntry> evaluate(Predicate predicate);
 
-    /**
```

Comment:
I kept the methods in `IndexStore` as they are used in there. Might make sense to refactor that as well (they are used to implement other methods of the interface only) and for `getRecords` implementation which seems to be used by PredicateAPI.